src/main.rs

```diff
@@ -437,6 +446,8 @@ impl Args {
                 };
 
                 let no_drop_root = submatches.occurrences_of("no-drop-root") == 1;
+                let native_profiling = submatches.occurrences_of("native-profiling") == 1;
```

Comment:
Might be able to use `is_present` here like we do for the boolean options below.